locales/pt-BR/translation.json

```
@@ -0,0 +1,57 @@
+{
+    "Workday Waiver Manager": "Gerenciador de Dia Livre",
+    "Exit": "Sair",
+    "Cut": "Cortar",
+    "Copy": "Copiar",
+    "Paste": "Colar",
+    "Select All": "Selectionar Tudo",
+    "Preferences": "Preferências",
+    "Migrate to flexible calendar": "Migrar para calendário flexível",
+    "Export database": "Exportar base de dados",
+    "Import database": "Importar base de dados",
+    "Clear database": "Limpar base de dados",
+    "Reload": "Recarregar",
+    "Toggle Developer Tools": "Alternar Ferramentas de Desenvolvimento",
```

Desenvolvedor

---

I want to avoid terms that are not gender-neutral.
What about?

```suggestion
    "Toggle Developer Tools": "Alternar Ferramentas de Desenvolvedores",
```

---

I searched on chrome help and it seems the Portuguese page calls it "DevTools" even in Portuguese. Maybe that makes sense here?

---

https://developers.google.com/web/tools/chrome-devtools?hl=pt-br